src/js/angular/aclmanagement/model.js

```diff
@@ -16,7 +16,7 @@ export class ACListModel {
 
 
     size(scope) {
-        return this._aclRules.get(scope).length;
+        return this._aclRules.get(scope) && this._aclRules.get(scope).length;
```

Comment:
And what value type will this return if there are no rule for that scope?

Reply:
added comment and return types